modules/heat_transfer/doc/content/source/kernels/ADHeatConduction.md

```diff
@@ -2,14 +2,19 @@
 
 ## Description
 
-`ADHeatConduction` is the implementation of the heat diffusion equation in [HeatConduction](/HeatConduction.md) within the framework of automatic differentiation.
-The `ADHeatConduction` kernel implements the heat equation given by Fourier's Law where The heat flux is given as
-\begin{equation}
-\mathbf{q} = - k\nabla T,
-\end{equation}
-where $k$ denotes the thermal conductivity of the material. $k$ can either be an `ADMaterial` or traditional `Material`.
-
-This class inherits from the [ADDiffusion](/ADDiffusion.md) class.
+`ADHeatConduction` is the implementation of [HeatConduction](/HeatConduction.md) within the framework of [!ac](AD). Please see the [HeatConduction](/HeatConduction.md) documentation for more information.
+
+For this object, the diffusion coefficient can either be an `ADMaterial` or traditional `Material`, though the Jacobian will be perfect if using an `ADMaterial`.
```

Comment:
```suggestion
For this object, the [!param](Kernels/ADHeatConduction/thermal_conductivity) must be an AD material property. 
```

Reply:
There is really no such thing as an `ADMaterial`. Any given `Material` may define both non-AD and AD material properties